### PR TITLE
feat(ui): Add `story-ui` command to run internal storybook

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Adds a new `yarn story-ui` command that builds ONLY the internal sentry storybook stories.

### CODE REVIEW

File: static/app/views/stories/storyList.tsx
Comment (ryan953) [inline static/app/views/stories/storyList.tsx:1]: seems unrelated to the build stuff but w/e

**Review by ryan953 — APPROVED** [review]

File: static/app/views/stories/storyList.tsx
Comment (evanpurkhiser) [reply static/app/views/stories/storyList.tsx:1]: It is related, it's no longer at `/stories/` now

Comment (ryan953) [reply static/app/views/stories/storyList.tsx:1]: oh, derp, nice!

**Comment by ryan953** [comment]
Whatever the strategy is, we should make it work for getsentry too. I'm worried that splitting this off onto vercel has low value and cut off all those components from participating. We should describe the value of this PR instead of briefly restarting what the code states

**Review by ryan953 — CHANGES_REQUESTED** [review]
Whatever the strategy is, we should make it work for `getsentry` too. I'm worried that splitting this off onto vercel has low value on it's own, and will prevent off all those components from participating which I think is more interesting.
We should describe the value of this PR instead of briefly restarting what the code states in the description.

**Pull request closed without merging**